### ISSUE TITLE
Bug with file permissions on Kali

### ISSUE THREAD

**Issue by xavier2dc** [Corb3nik/Talk-MontreHack-Homework#1]
When deploying the challenge on Kali (2017.1), the file `custom.cnf` has world-writable permissions:
```
root@262833a44f83:/var/www/html# ls -l /etc/mysql/conf.d/           
total 8
-rwxrwxrwx 1 root root 31 Jul 19 22:49 custom.cnf
```

which prevents mysql from taking that file into account:
```
root@262833a44f83:/var/www/html# service mysql restart
[....] Stopping MySQL database server: mysqldWarning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
Warning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
. ok 
Warning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
[....] Starting MySQL database server: mysqldWarning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
Warning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
 .Warning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
Warning: World-writable config file '/etc/mysql/conf.d/custom.cnf' is ignored
. ok 
[info] Checking for tables which need an upgrade, are corrupt or were 
not closed cleanly..
```
Then the challenge becomes unsolvable :(

Adding `RUN chmod 444 /etc/mysql/conf.d/custom.cnf` in `challenge/Dockerfile`, then `sudo docker-compose build web` solves this issue.

**Comment by Corb3nik** [comment]
Interesting, haven't had this issue on OSX.
I'll fix this up soon.

Thanks

**Comment by Corb3nik** [comment]
Should be fixed by PR #2
